### ISSUE TITLE
Broken data in metadata result

### ISSUE THREAD

**Issue by Fuchsoria** [Borewit/music-metadata-browser#740]
Hi, today we found that metadata parse doesn't work anymore in our case, 
![image](https://user-images.githubusercontent.com/43413472/155300548-6f1372c9-2126-4619-bc17-91b7007ed531.png)

package versions were tested:  2.2.6, 2.5.4.
browser chrome latest version

```
const metadata = mm.parseReadableStream(file.stream());

metadata.then(result => {
    console.log(result);
});
```

**Comment by mhyeganeh** [comment]
Same Here:
![Annotation 2022-02-24 164512](https://user-images.githubusercontent.com/32367934/155531004-2cbdc02c-71dd-46fc-acbc-4b019b5d2e04.jpg)

```
await musicMetadata.parseBlob(audio_file).then(metadata => {
     console.log(metadata);
});
```
I was using this package and top method for a long time and suddenly it stopped working! (Aroung 2 weeks ago...)

**Comment by Borewit** [comment]
Thank reproducing the issue, that allowed me to nail this issue!

Should be resolved in [music-metadata-browser v2.5.5](https://github.com/Borewit/music-metadata-browser/releases/tag/v2.5.5) my friend [mhyeganeh](https://github.com/mhyeganeh).

**Comment by mhyeganeh** [comment]
@Borewit 

Good Job! 👍 and so many thanks for your effort. It has benn resoved completely in [2.5.5](https://github.com/Borewit/music-metadata-browser/releases/tag/v2.5.5)
